ml-notebook/environment.yml

```diff
@@ -8,6 +8,7 @@ dependencies:
  - cuda-version>=12.0
  - flax>=0.8.0
  - jax
+ - jaxlib>=0.4.23=cuda120*
```

Comment:
This pin on using a `cuda120*` build of jaxlib might actually need to go away if we're planning on adding multi-arch builds of `ml-notebook` (see https://github.com/pangeo-data/pangeo-docker-images/pull/399#pullrequestreview-2143121202 for context), but will deal with this later since the same pin is present on Tensorflow.